### DISCUSSION TITLE
LEAK - Bendy's week being finished soon!

### DISCUSSION THREAD

**Discussion by Pico8430** [Pico8430/Indie-Cross-PE-Port#12] [Announcements]
# **_THE LEAK AND PROGRESS_**

We are still working on this port, and we are trying our best! Since i don't want development to be inactive or dead, here's a leak that were actually making some progress!

# 

https://user-images.githubusercontent.com/91833725/166124691-4cffc693-cc30-4b18-b074-e717bedb144b.mp4

#

**Comment by DaveFan19** [comment]
niiiice!

**Comment by CreativeUsernameAttempt** [comment]
I feel like you should fix bf being the demo one
